### PR TITLE
Clarify OSS definition for vendors

### PR DESCRIPTION
**Pull request by cartermp**
This came up in DMs regarding Grafana, who have both an open source product and a closed source cloud offering.

### CODE REVIEW

File: content/en/ecosystem/vendors.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ Some organizations provide a [distribution](/ecosystem/distributions/) (of
 customized OpenTelemetry components), that provides additional capabilities or
 for improved ease of use.
 
+Open Source (OSS) refers to a vendor who has an observability product
```
Comment (theletterf) [inline content/en/ecosystem/vendors.md:15]:
```suggestion
Open Source (OSS) refers to vendors who maintain an observability product
```

File: content/en/ecosystem/vendors.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ Some organizations provide a [distribution](/ecosystem/distributions/) (of
...
+that is [open source](https://opensource.org/osd). The vendor may still have
+other products that are closed source, such as a SaaS offering that
+hosts an open source product for their customers.
```
Comment (theletterf) [inline content/en/ecosystem/vendors.md:18]:
```suggestion
is [open source](https://opensource.org/osd). The vendors may still have
other products that are closed source, such as SaaS offerings that
host open source products for their customers.
```

**Review by theletterf — APPROVED** [review]

**Review by chalin — COMMENTED** [review]
Hmm, sorry @theletterf, but I prefer @cartermp's original.

That being said, I think that we can do even better by lifting up footnote 1, and folding it into the explanation. WDYT?

**Comment by theletterf** [comment]
@chalin That's OK! I think the singular can be a bit confusing, but overall the concept is clear. Merging!

**Pull request merged**